plugin.xml

```diff
@@ -64,12 +79,21 @@ Features
          <tag>smartphone</tag>
          <tag>configuration</tag>
          <tag>android</tag>
+         <tag>ios</tag>
       </fr>
       <en>
          <tag>inventaire</tag>
```

Comment:
Inventaire is the french translation, it should be inventory

In line 78 is the opposite